index.rst

```diff
@@ -44,14 +44,579 @@
+   If the capabilites were in place, then the on-the-fly users would certainly take advantage of them.
+
+FAFF-REQ-014
+^^^^^^^^^^^^
+
+**Specification:** The camera visualization tool shall support being deployed in places where historical data is available.
+
+**Rationale:** This will need to be available to people using the `RSP`_ and commissioning cluster.
+The general commissioning / SV use case is to be able to examine aspects of image quality that cross detector boundaries (e.g., stray and scattered light, satellite trails, pervasive issues across detectors) for which full focal plane visualization is critical. 
+Of course, these studies will involve looking at images that date back in time, and therefore will need to be executed from the RSP (or other processing center).
+This could be useful even for summit operations if it allows display of historic images (for comparison with new images).
+The historical data on the summit is currently limited to 30 days.
+
+**Priority: 1**
+
+**Current shortcomings:** Firefly may not meet all of the requirements for all image visualization, specifically in regards to full-frame visualization.
+Deployment not nested into current `RSP`_ deployment strategy.
+It requires a mechanism to locate the data for a given obsid, but this is also presumably be possible. 
+
+**Applicable Use-cases:** Rapid per sensor image display and inspection.
+
+**Suggested Implementation to fulfill requirement:** 
+Deploy the camera visualization tool and Bokeh apps as part of the standard `RSP`_ packaging.
+Installing the camera image visualization server at the USDC (SLAC) is certainly feasible. 
+
+
+FAFF-REQ-026
+^^^^^^^^^^^^
+
+**Specification:** The display tool should be able to display data obtained from the butler, or obtained from a users interactive Jupyter session
+
+**Rationale:** Displaying images with full DM ISR applied, co-added images etc.
+This is required to perform much of the post on-the-fly analysis during commissioning.
+
+
+**Priority:** 1
+
+**Current shortcomings:** DM has an abstract image visualization interface (afw). 
+Needs to be evaluated to see if this could be used to meet all the requirements.
+
+**Applicable Use-cases:** 
+
+**Suggested Implementation to fulfill requirement:** 
+
+FAFF-REQ-036
+^^^^^^^^^^^^
+
+**Specification:** Ability to overlay markings at user-provided pixel positions
+
+**Rationale:** Used to indicate which sources are used in PSF analysis, blends, from catalogs etc.
+
+**Priority:** 1
+
+**Current shortcomings:** Currently unable to interface to DM (essentially the butler, pre-req is FAFF-REQ-026_) 
+
+**Applicable Use-cases:**
+
+**Suggested Implementation to fulfill requirement:**
+
+
+FAFF-REQ-017
+^^^^^^^^^^^^
+
+**Specification:** Ability to choose between minimal ISR versus some more sophisticated ISR (for example, the calexp images served from a butler)
+
+**Rationale:** 
```

Comment:
Suggest: "For general SV purposes, it should be possible to inspect images at various stages of data processing both to evaluate the ISR performance and to visualize more generic representations of pixel-level information (e.g., mask planes)"